Intersect.Server.Core/Maps/MapInstance.cs

```diff
@@ -924,7 +931,7 @@ private void SpawnAttributeItem(int x, int y)
             {
                 mapItem.Quantity = 1;
             }
-            AddItem(mapItem);
+            AddItem(null, mapItem);
```

Comment:
The one misgiving I have about doing this is because I honestly want to create a `MapItemSource` class that refers to the map instance (again via map instance ID and `WeakReference<MapInstance>`) since it's still valuable information.